builder/docker/tinygo/.tinygo-ver

```diff
@@ -1 +1 @@
-v0.22.0
\ No newline at end of file
+v0.22.0
```

Comment:
Shouldn't this be v0.23.0?

Reply:
This PR is only for whats in main `v0.22.0`. That green line is just my auto formatting adding a newline haha. 

There's a seperate PR for `v0.23.0` which doesn't build yet 😅

Reply:
Gotcha :P Proceed